jaxb-ri/runtime/impl/src/main/jdk9/module-info.java

```diff
@@ -58,9 +57,11 @@
     requires transitive java.activation;
     requires transitive java.xml;
 
-    exports com.sun.istack;
-    exports com.sun.istack.localization;
-    exports com.sun.istack.logging;
+    requires txw2;
```

Comment:
These requires (txw2, istack, etc) are copied by dependency plugin to module path. They will not be available for jaxb-ri.zip bundle, which has its classes repackaged.